### PR TITLE
Use absolute namespace

### PR DESCRIPTION
**Pull request by LukasReschke**
Ref https://bugs.php.net/bug.php?id=66773

### CODE REVIEW

File: apps/files_external/lib/sftp.php [hunk 1]
```diff
@@ -30,8 +30,6 @@
  */
 namespace OC\Files\Storage;
 
-use phpseclib\Net\RSA;
```
Comment (LukasReschke) [inline apps/files_external/lib/sftp.php]: Remark: Was never used in this class.

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by Xenopathic** [comment]
:+1:

I'm amazed that the original code worked at all.

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**